src/library/FIFO.js

```diff
@@ -1,12 +1,10 @@
 /**
- * A First-In-First-Out (FIFO) class that maintains a queue of
- * elements.  New elements are added to the tail, and the oldest
- * element is removed from the head when the FIFO is full.  If the
- * size of the queue is 1, it will always contain the newest element
- * that was inserted.
+ * A First-In-First-Out (FIFO) class that only design for vu-meter to
+ * maintains a queue of elements. In class does not has pull,
+ * getBufferLength function. It includes the getMinValue function for
+ * tracking the minimum value in the FIFO class.
```

Comment:
Perhaps:

`@classdesc A FIFO (First-In-First-Out) class that is specifically designed for the VUMeter class in the library directory. This class does not have a method to get or drop a head value, but it includes one for getting a minimum value form the queue.`

https://jsdoc.app/tags-classdesc.html: this tag seems better for this purpose.